src/main/java/network/brightspots/rcv/CsvCvrReader.java

```diff
@@ -1,6 +1,6 @@
 /*
  * RCTab
- * Copyright (c) 2017-2023 Bright Spots Developers.
+ * Copyright (c) 2017-2022 Bright Spots Developers.
```

Comment:
Sorry, this may seem like a dumb change, but it happened due to the way `BrightSpotsCopyrightProfile.xml` is set up. I have an open issue to update this, and will update all other files at the same time.